x/blob/types/payforblob.go

```diff
@@ -45,12 +45,12 @@ const (
 // See: https://github.com/cosmos/cosmos-sdk/blob/v0.46.15/docs/building-modules/messages-and-queries.md#legacy-amino-legacymsgs
 var _ legacytx.LegacyMsg = &MsgPayForBlobs{}
 
-func NewMsgPayForBlobs(signer string, blobs ...*blob.Blob) (*MsgPayForBlobs, error) {
+func NewMsgPayForBlobs(signer string, version uint64, blobs ...*blob.Blob) (*MsgPayForBlobs, error) {
```

Comment:
[optional][follow-up] this param could clarify that it's `appVersion` and not `shareVersion` or `namespaceVersion`

Reply:
Good point. I think that makes sense